homeworks/ovarikristof/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,14 @@
+full_name = (input("Please enter your full name."))         #BEKÉRJÜK A TELJES NEVET.
+
+age = (str(input("Please enter your age.")))                #BEKÉRJÜK A KORÁT.
+
+experince_in_python = (str(input("Please enter your experience with python in years.")))    #BEKÉRJÜK, HOGY HÁNY ÉV TAPSZTALATA VAN PYTHON-BAN.
```

Comment:
itt is ugyanez igaz, sem a külső zárójel, se a str() nem szükséges. Mivel ez egy szám, egy int() típuskonverzió viszont hasznos lehet!